File: linearSearch/linearSearch.java
public class linearSearch {
    public static int arr[] = {-5,2,10,4,6};
    public static void main(String args[]){
        System.out.println(lineSearch(arr, 10));
    }

    public static int lineSearch(int n[], int t){
        for(int i = 0; i < n.length; i++){
            if(n[i] == t){
                return i;
            }
        }
        return -1;
    }
}

File: linearSearch/linearSearch.js
let arr = [-5,2,10,4,6];

function linearSearch(n, t){
    for(let i = 0; i < n.length; i++){
        if (n[i] === t){
            return i;
        }
    }
    return -1;
}

console.log(linearSearch(arr, 30));